apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -111,221 +101,4 @@ const Instructions: React.FunctionComponent<InstructionsProps> = ({
     />
   );
 };
-
-interface InstructionsPanelProps {
-  /** Primary instructions text to display. */
-  text: string;
-  /** Optional message to display under the main text. This is typically a validation message. */
-  message?: string;
-  /** Key for rendering the optional message. A unique value ensures the appearance animation shows. */
-  messageIndex?: number;
-  /** If the instructions panel should be rendered vertically or horizontally. Defaults to vertical. */
-  layout?: 'vertical' | 'horizontal';
-  /** Display theme. Defaults to Dark. */
-  theme?: 'Dark' | 'Light';
-  /**
-   * A callback when the user clicks on clickable text.
-   */
-  handleInstructionsTextClick?: (id: string) => void;
-  predictSettings?: LevelPredictSettings;
-  predictResponse?: string;
-  setPredictResponse: (response: string) => void;
-  predictAnswerLocked: boolean;
-  /** Optional classname for the container */
-  className?: string;
-  offerBrowserTts?: boolean;
-  canShowNextButton: boolean;
-  hasNextLevel: boolean;
-  useSecondaryFinishButton: boolean;
-  onContinueOrFinish: () => void;
-  bottomComponent?: React.ReactNode;
-}
-
-/**
- * Renders the instructions panel view. This was initially set up as a separate component
- * so that it could be used without the Lab2 redux integration if necessary.
- * If the level is a predict level, the predict reset button now uses redux, as it needs
- * multiple unique redux values and there isn't a clear use case for having no redux integration
- * anymore.
- * TODO: Determine if we need this separate component anymore, or if we can merge this into Instructions.
```

Comment:
Well, I found a use for this separate component! I don't totally understand the comment about predict levels and the predict reset button, if anyone could help me understand if that comment is still relevant (I removed it, but can re-add).

Reply:
Yeah the original intent of splitting it was to have a component with no redux dependencies. When I added predict levels, it started to have a redux dependency if it's a predict level. That's probably not relevant for the preview case?

Reply:
Ahh got it got it -- I'm getting away with it because Redux is used in the `PredictSummary` component, which isn't rendered in the preview case.